src/generic_ui/polymer/cloud-install.html

```diff
@@ -58,13 +63,39 @@
     <core-signals on-core-signal-open-cloud-install-dialog='{{ openLoginDialog }}'></core-signals>
 
     <!-- TODO: use core-pages rather than multiple dialogs -->
+    <uproxy-dialog layered='false' backdrop='true' id='regionDialog'>
+      <h1>Create a cloud server</h1>
+      <core-icon icon='clear' on-tap='{{ closeDialogs }}'></core-icon>
+      <p>We can help you create a uProxy cloud server in a country of your choice.</p>
+      <p>To get started, please select the region for your server. You should choose the region closest to you to minimize latency.</p>
+      <div style='text-align:center;'>
+        <paper-dropdown-menu label='Select Region'>
```

Comment:
The popup causes the dialog to show a scrollbar when it appears. Have you considered a fixed size component, e.g. radio buttons? I expect a small number of options (probably no need to have two US options, for example).